src/exporters/ImageExporter.js

```diff
@@ -60,7 +60,8 @@ class ImageExporter {
                         dialog.dismiss();
                     }
                 }
-            ]
+            ],
+            skipHidePreviousOverlay: true
```

Comment:
👍

Reply:
as per discussion with @akhenry, we can have a better solution by,

1. add a flag inside Overlay.js `this.autoHide = options.autoHide;`            // autoHideOnNewOverlay default true.
2. change logic inside `showOverlay` method in OverlayAPI.js to add class 'invisible' based on it
3. finally  use it (`autoHide: false`) when we show preview/view large.